client-course-schedulizer/src/utilities/helpers/teamMembers.ts

```diff
@@ -0,0 +1,49 @@
+export interface TeamMember {
+  bio: string;
+  name: string;
+  photo: string;
+  website: string;
+}
+
+/* Basic information about the project team members */
+export const team: TeamMember[] = [
+  {
+    bio:
+      "Computer Science student at Calvin University. Jonathan is a hard worker, a faster runner, and has previous web development experience.",
+    name: "Jonathan Ellis",
+    photo:
+      "https://avatars3.githubusercontent.com/u/49655167?s=460&u=b81f525db7da58b19bc28588282b366cd8b748ab&v=4",
+    website: "https://github.com/Jonri2",
+  },
+  {
+    bio:
+      "Computer Science and Mathematics at Calvin University. Bryant hoping to go into software development while keeping Mathematics as a lifelong hobby.",
+    name: "Bryant George",
+    photo:
+      "https://avatars1.githubusercontent.com/u/51130302?s=460&u=83737db9aeff2b377654a3e0a7cd3dc39f54f7ff&v=4",
+    website: "https://github.com/Syobnaf",
```

Comment:
I've recently changed my GitHub username:
```suggestion
    website: "https://github.com/bryantgeorge",
```